api/tests/integration/ref/formats/ket_cdxml_ket.py.out

```diff
@@ -81,15 +81,15 @@ agents.ket
+{"root":{"nodes":[{"$ref":"mol0"},{"$ref":"mol1"},{"$ref":"mol2"},{"type":"arrow","data":{"mode":"open-angle","pos":[{"x":8.5,"y":-5.074999809265137,"z":0.0},{"x":12.800000190734864,"y":-5.025000095367432,"z":0.0}]}},{"type":"text","data":{"content":"{\"blocks\":[{\"text\":\"test\",\"inlineStyleRanges\":[{\"offset\":0,\"length\":4,\"style\":\"CUSTOM_FONT_SIZE_14px\"}],\"entityRanges\":[],\"data\":{}}],\"entityMap\":{}}","position":{"x":10.149999618530274,"y":-5.574999809265137,"z":0.0}}}]},"mol0":{"type":"molecule","atoms":[{"label":"C","location":[6.520544052124023,-5.325058937072754,0.0]},{"label":"C","location":[6.520544052124023,-6.325052738189697,0.0]},{"label":"C","location":[5.654489040374756,-6.82504940032959,0.0]},{"label":"C","location":[4.788554668426514,-6.324931621551514,0.0]},{"label":"C","location":[4.788554668426514,-5.32493782043457,0.0]},{"label":"C","location":[5.654489040374756,-4.824941158294678,0.0]},{"label":"C","location":[5.654489040374756,-3.8249473571777345,0.0]},{"label":"O","location":[4.788554668426514,-3.324950695037842,0.0]},{"label":"C","location":[6.520544052124023,-3.324950695037842,0.0]}],"bonds":[{"type":1,"atoms":[0,1]},{"type":2,"atoms":[1,2]},{"type":1,"atoms":[2,3]},{"type":2,"atoms":[3,4]},{"type":1,"atoms":[4,5]},{"type":2,"atoms":[0,5]},{"type":1,"atoms":[5,6]},{"type":2,"atoms":[6,7]},{"type":1,"atoms":[6,8]}]},"mol1":{"type":"molecule","atoms":[{"label":"C","location":[17.261444091796876,-5.758026123046875,0.0]},{"label":"C","location":[17.261444091796876,-6.75801944732666,0.0]},{"label":"C","location":[16.395389556884767,-7.258016109466553,0.0]},{"label":"C","location":[15.529335975646973,-6.75801944732666,0.0]},{"label":"C","location":[15.529335975646973,-5.758026123046875,0.0]},{"label":"C","location":[16.395389556884767,-5.258028984069824,0.0]},{"label":"C","location":[16.395389556884767,-4.258035659790039,0.0]},{"label":"C","location":[15.529335975646973,-3.7580387592315676,0.0]},{"label":"C","location":[14.663400650024414,-3.2580418586730959,0.0]},{"label":"O","location":[16.02933120727539,-2.8919837474823,0.0]},{"label":"C","location":[15.029337882995606,-4.624094009399414,0.0]}],"bonds":[{"type":1,"atoms":[0,1]},{"type":2,"atoms":[1,2]},{"type":1,"atoms":[2,3]},{"type":2,"atoms":[3,4]},{"type":1,"atoms":[4,5]},{"type":2,"atoms":[0,5]},{"type":1,"atoms":[5,6]},{"type":1,"atoms":[6,7]},{"type":1,"atoms":[7,8]},{"type":1,"atoms":[7,9]},{"type":1,"atoms":[7,10]}]},"mol2":{"type":"molecule","atoms":[{"label":"Cl","location":[10.475000381469727,-4.5,0.0]}],"bonds":[]}}
```

Comment:
make ident for json and move output into a separate file

Reply:
solved